Temperature_API/crud.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+
+import httpx
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from City_CRUD_API.models import City
+from Temperature_API import models
+from Temperature_API.models import Temperature
+from Temperature_API.schemas import TemperatureCreate
+
+
+WEATHER_API_URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = "d0eae92abc5b4efaa76144848242007"
```

Comment:
Storing API keys directly in the code is not a good practice. Consider using environment variables or a configuration file to manage sensitive information like API keys.